arangod/ClusterEngine/ClusterEngine.cpp

```diff
@@ -77,6 +77,16 @@ ClusterEngine::ClusterEngine(application_features::ApplicationServer& server)
 
 ClusterEngine::~ClusterEngine() = default;
 
+void ClusterEngine::setActualEngine(StorageEngine* e) {
+  _actualEngine = e;
+  if (isMMFiles()) {
+    LOG_TOPIC("6ab73", WARN, arangodb::Logger::STARTUP)
+        << "The MMFiles storage engine is deprecated starting with ArangoDB "
+           "v3.6.0 and will be removed in a future version. "
+        << "Please plan for a migration to the RocksDB engine.";
```

Comment:
Does it make any difference whether one or all lines use `<<` here?

Reply:
Does not matter. With `<<` it is one cheap operation more at startup. The only thing to avoid is `std::endl` && `std::flush` because flushing is expensive and the former contains a flush.